src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,40 @@
-export const Tabs = () => {};
+export const Tabs = (
+  { tabs, selectedTabId, onTabSelected },
+) => {
+  const TabClick = (tab) => {
```

Comment:
we usually name function and vars in camelCase
```suggestion
  const handleTabClick = (tab) => {
```